src/plugins/recursion/README.md

```diff
@@ -23,25 +23,37 @@ Users have to define recursive setting specifications in the following way:
 
 ```
 kdb mount config.dump /recursive dump recursion
-kdb setmeta user/recursive check/recursion/vertex "menu, submenu"
-kdb setmeta user/recursive check/recursion/leaf "param, menupoint"
+kdb setmeta user/recursive check/recursion/vertex "A, B"
+kdb setmeta user/recursive check/recursion/leaf "z, y"
 ```
 
-`check/recursion/vertex` basically means that keys containing `menu` or `submenu` after `/recursive`
-will have further configurations below them. So users would build up structures like this:
+`check/recursion/vertex` basically means that keys containing `A` or `B` after `/recursive`
```

Comment:
The word "basically" is useless. Say it like it is. And again: with+without example.